### PR TITLE
Improve operators for onnxruntime

### PR DESCRIPTION
**Pull request by scxiao**
change squeeze and unsqueeze to support axis to be negative. Also, remove unnecessary multibroadcast in parsing onnx file.

### CODE REVIEW

File: test/cpu_ops_test.cpp [hunk 1]
```diff
@@ -324,6 +324,19 @@ TEST_CASE(squeeze_test)
...
+    {
+        migraphx::program p;
+        std::vector<float> data(4 * 3 * 3);
+        migraphx::shape s1{migraphx::shape::float_type, {4, 1, 3, 1, 3}};
+        migraphx::shape s2{migraphx::shape::float_type, {4, 1, 3, 3}};
+        auto l0 = p.add_literal(migraphx::literal{s1, data});
+        p.add_instruction(migraphx::op::squeeze{{-2}}, l0);
+        p.compile(migraphx::cpu::target{});
+        auto result = p.eval({});
+        EXPECT(result.get_shape() == s2);
```
Comment (pfultz2) [inline test/cpu_ops_test.cpp]: Can we move this test to op_shape_test.cpp?

File: test/cpu_ops_test.cpp [hunk 2]
```diff
@@ -361,6 +374,18 @@ TEST_CASE(unsqueeze_test)
...
+    {
+        migraphx::program p;
+        std::vector<float> data(4 * 3 * 3);
+        migraphx::shape s1{migraphx::shape::float_type, {4, 3, 3}};
+        migraphx::shape s2{migraphx::shape::float_type, {4, 3, 1, 3}};
+        auto l0 = p.add_literal(migraphx::literal{s1, data});
+        p.add_instruction(migraphx::op::unsqueeze{{-2}}, l0);
+        p.compile(migraphx::cpu::target{});
+        auto result = p.eval({});
+        EXPECT(result.get_shape() == s2);
```
Comment (pfultz2) [inline test/cpu_ops_test.cpp]: Can we move this test to op_shape_test.cpp?

File: test/cpu_ops_test.cpp (hunk 1 above)
Comment (scxiao) [reply test/cpu_ops_test.cpp]: moved

File: test/cpu_ops_test.cpp (hunk 2 above)
Comment (scxiao) [reply test/cpu_ops_test.cpp]: moved

File: src/targets/gpu/device/logsoftmax.cpp [hunk 3]
```diff
@@ -11,9 +11,10 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 namespace device {
 
-void logsoftmax(hipStream_t stream, const argument& result, const argument& arg, int axis)
+void logsoftmax(hipStream_t stream, const argument& result, const argument& arg, int64_t axis)
```
Comment (pfultz2) [inline src/targets/gpu/device/logsoftmax.cpp]: The axis should be computed in the operator and passed to `logsoftmax` device function.

Comment (scxiao) [reply src/targets/gpu/device/logsoftmax.cpp]: fixed accordingly

**Pull request merged**